pkg/storegateway/bucket.go

```diff
@@ -907,13 +948,12 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 				return
 			}
 		}
-		if set.Err() != nil {
-			err = status.Error(codes.Unknown, errors.Wrap(set.Err(), "expand series set").Error())
+		if seriesSet.Err() != nil {
+			err = errors.Wrap(seriesSet.Err(), "expand series set")
 			return
 		}
-		mergeDuration := time.Since(begin)
-		mergeStats.mergeDuration += mergeDuration
-		s.metrics.seriesMergeDuration.Observe(mergeDuration.Seconds())
```

Comment:
Has this changed intentionally? Logic is the same, but I've the feeling this has changed unintentionally.

Reply:
i was changed and then changed back.. and in the meantime it got a new form. i will revert it